lib/tests/it/main.rs

```diff
@@ -306,7 +306,10 @@ fn validate_tar_v1<R: std::io::Read>(mut src: tar::Archive<R>) -> Result<()> {
     .into_iter()
     .map(Into::into);
 
-    let expected = prelude.chain(common_tar_structure());
+    let content = [("usr", Directory, 0o755), ("boot", Directory, 0o755)];
```

Comment:
I also did cross-verify that this test fails when we don't include the corresponding changes to `export.rs`, and I just looked at the tarballs generated by `cargo run -- internal-only-for-testing create-fixture` before and after.